### PR TITLE
UI/Responsive for Induction Flow

### PR DESCRIPTION
**Pull request by brandonfancher**
I *think* this gets us where we need to be with regard to styles/responsiveness. 🤞

## What's in it?
* Style and make responsive the entire induction flow from the perspective of 1) logged-out users, 2) invitees, 3) inviters and 4) witnesses/endorsers.
* Progress/Step indicator always shows you where you are in the process.
* Reusing member card and member profile components from member pages as the new-member-preview components in this flow.

## Future PRs
* Show endorser full names instead of only EOS names. (#95)
* This is a bit of a bowl of spaghetti. We need to spend some time refactoring after initial launch.

## Screenshots
(but you'll need to pull it down and fire it up to see it all)
### Invite
image
image
image

### Profile
image
image

### CODE REVIEW

File: packages/webapp/src/inductions/components/induction-journey-container.tsx [hunk 1]
```diff
@@ -18,14 +18,42 @@ export const INVITEE_INDUCTION_STEPS: Step[] = [
     },
     {
         title: "DONATE",
-        text: `${minimumDonation} to the Eden community.`,
+        text: `Give ${assetToString(
+            minimumDonationAmount
+        )} to the Eden community.`,
```
Comment (sparkplug0025) [inline packages/webapp/src/inductions/components/induction-journey-container.tsx:23]: Unless you really want to show 10.00000000 WAX, you can use `assetToString(minimumDonationAmount, 2)` -- which will limit in 2 decimals

File: packages/webapp/src/inductions/components/induction-journey-container.tsx [hunk 2]
```diff
@@ -18,14 +18,42 @@ export const INVITEE_INDUCTION_STEPS: Step[] = [
...
+    {
+        title: "INDUCT & ENDORSE",
+        text:
+            "Record and attach induction ceremony. Inviter and witnesses endorse invitee.",
+    },
+    {
+        title: "INVITEE DONATION",
+        text: `Invitee donates ${assetToString(
+            minimumDonationAmount
+        )} to the Eden community.`,
```
Comment (sparkplug0025) [inline packages/webapp/src/inductions/components/induction-journey-container.tsx:49]: same as above

File: packages/webapp/src/inductions/components/induction-profile-form.tsx [hunk 3]
```diff
@@ -78,7 +82,7 @@ export const InductionProfileForm = ({
             </Form.LabeledSet>
 
             <Form.LabeledSet
-                label="Attributions"
+                label="Profile image attribution (optional)"
```
Comment (sparkplug0025) [inline packages/webapp/src/inductions/components/induction-profile-form.tsx:85]: Will people understand that attribution = copyrights?

File: packages/webapp/src/inductions/components/induction-step-endorsement.tsx [hunk 4]
```diff
@@ -155,128 +171,156 @@ export const InductionStepEndorsement = (props: Props) => {
...
-            is endorsed and it's only waiting for his donation to complete the
-            induction!
-        </div>
+        <Text>
+            This induction is fully endorsed! As soon as the prospective member
+            completes their donation to the Eden community, their membership
+            will be activated and their Eden NFTs will be minted and
+            distributed.
+        </Text>
     );
```
Comment (sparkplug0025) [inline packages/webapp/src/inductions/components/induction-step-endorsement.tsx:325]: great job on this! oh man, I'm really missing your screenshots, I just need to use my imagination to see how better it is now :(

**Review by sparkplug0025 — APPROVED** [review]
👏 eager to see this!

File: packages/webapp/src/inductions/components/induction-journey-container.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/inductions/components/induction-journey-container.tsx:23]: It has assetToString in there.

Comment (brandonfancher) [reply packages/webapp/src/inductions/components/induction-journey-container.tsx:23]: Oh, the 2! Missed that. Thank you!

File: packages/webapp/src/inductions/components/induction-profile-form.tsx (hunk 3 above)
Comment (brandonfancher) [reply packages/webapp/src/inductions/components/induction-profile-form.tsx:85]: It’s a good question. I think so. I’ll give it some more thought. I am consciously steering clear of the word copyright though.

File: packages/webapp/src/inductions/components/induction-step-endorsement.tsx (hunk 4 above)
Comment (brandonfancher) [reply packages/webapp/src/inductions/components/induction-step-endorsement.tsx:325]: Thanks!

File: packages/webapp/src/inductions/components/induction-journey-container.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/inductions/components/induction-journey-container.tsx:23]: `assetToString` defaults to 2 decimals. (And it's already truncating properly.)

**Pull request merged**